services/adminService.js

```diff
@@ -53,29 +56,67 @@ exports.deleteUsers = async (users, isForceDelete) => {
     if (!user) {
       throw new ApiError(404, 'User not found')
     }
-    const reservation = await Reservation.find({ user: id })
-    if (reservation.length > 0 && isForceDelete) {
+    const reservations = await Reservation.find({ user: id })
+    if (reservations.length > 0 && isForceDelete) {
       await Reservation.deleteMany({ user: id })
+      reservations.forEach(async ({ user, hotel }) => {
+        const { name } = await Hotel.findById(hotel)
+        notifyUser(
+          user.isSmsAllowed,
+          user.email,
+          'Cancelled reservation',
+          'reservationRemoved',
+          user.firstName,
+          name,
+          'BookingCloneApi',
+          user.phoneNumber,
+          'Your reservation has been cancelled'
+        )
+      })
     }
-    if (reservation.length > 0 && !isForceDelete) {
+    if (reservations.length > 0 && !isForceDelete) {
       throw new ApiError(400, 'Remove reservations first')
     }
     await User.findByIdAndDelete(id)
+    notifyUser(
```

Comment:
are we going to notify user twice. Instead we can simply send one email (see example in a previous comment) 

possible solution (remember about TESTS)
![image](https://user-images.githubusercontent.com/38506213/110431871-a40e5680-80ae-11eb-8083-d57de101b73d.png)


add spaces between sentences (I forgot to add one)